platform/posix/ota_pal/source/ota_pal_posix.c

```diff
@@ -433,17 +434,35 @@ OtaPalStatus_t otaPal_CreateFileForRx( OtaFileContext_t * const C )
             {
                 /* POSIX port using standard library */
                 /* coverity[misra_c_2012_rule_21_6_violation] */
```

Comment:
Does the comment about MISRA violation now need to be moved to above the `fclose` call?

Reply:
Added in https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1551/commits/b99d4ab575f455287a26210bb5d10cf318aa3d03